Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o /out/linkerd2-proxy-init -mod=readonly -ldflags "-s -w" -v
 
 ## package runtime
-FROM debian:stretch-20190812-slim
+FROM debian:buster-20190910-slim
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
         iptables \
```

Comment:
Try adding

```
RUN update-alternatives --set iptables /usr/sbin/iptables-legacy \
    && update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
```

here to make current use of `iptables` work with buster.
(See https://github.com/linkerd/linkerd2/pull/4567 for more info.)